src/components/common/Dialogs/ManageWhitelistDialog/UserCheckbox/UserCheckbox.tsx

```diff
@@ -18,16 +17,17 @@ interface Props {
   checkedTooltipText?: string;
   unCheckedTooltipText?: string;
   showDisplayName?: boolean;
+  user?: User;
 }
 
 const UserCheckbox = ({
   walletAddress,
   name,
   checkedTooltipText,
   unCheckedTooltipText,
+  user,
   showDisplayName = true,
 }: Props) => {
-  const { user } = useUserByNameOrAddress(walletAddress);
```

Comment:
This component is only used for the Whitelist dialog and by the time it's rendered we'd have already fetched the users info, no need for a separate query